src/Go/boardAnalysis/goAI.ts

```diff
@@ -397,7 +397,8 @@ async function getIlluminatiPriorityMove(moves: MoveOptions, rng: number): Promi
 /**
  * Get a move that places a piece to influence (and later control) a corner
  */
-function getCornerMove(board: Board) {
+// eslint-disable-next-line @typescript-eslint/require-await
+async function getCornerMove(board: Board) {
```

Comment:
This doesn't make sense. You're disabling the lint warning for a useless async, but you're also *adding* `async` to this function in this change. Obviously, it worked fine without `async` before, so you should be able to revert this whole thing and lint shouldn't complain?